nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/state-management.xml

```diff
@@ -63,4 +63,68 @@
         <property name="Session Timeout">10 seconds</property>
         <property name="Access Control">Open</property>
     </cluster-provider>
+
+    <!--
+        Cluster State Provider that stores state in Redis. This can be used as an alternative to the ZooKeeper State Provider.
+
+        This provider requires the following properties:
+
+            Redis Mode - The type of Redis instance:
+                            - Standalone
+                            - Sentinel
+
+            Connection String - The connection string for Redis.
+                        - In a standalone instance this value will be of the form hostname:port.
+                        - In a sentinel instance this value will be the comma-separated list of sentinels, such as host1:port1,host2:port2,host3:port3.
+                        - In a clustered instance this value will be the comma-separated list of cluster masters, such as host1:port,host2:port,host3:port.
```

Comment:
I added the "Cluster" value to the Redis Mode property and stated how it is not currently supported for the state provider due limitations of Redis transactions in clustered mode.